config/routes.rb

```diff
@@ -3,4 +3,5 @@
   devise_for :users,
              controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }
   root to: 'home#index'
+  resources :users
```

Comment:
Show method also will be not used. So only :index ?

We will not show profile of user, because no specific information except name and email